testdata/configtest.json

```diff
@@ -319,7 +319,7 @@
    "baseCurrencies": "JPY",
    "assetTypes": "SPOT",
    "supportsAutoPairUpdates": false,
-   "pairsLastUpdated": 1557980263,
+   "pairsLastUpdated": 1997980263,
```

Comment:
Can revert these back to the original values as there's likely to be updates between now and 2033